Main.java

```diff
@@ -0,0 +1,15 @@
+public class Main{
+    public static void main(String[] args){
+        if(args.length>0){
+        for(String val: args)
+        System.out.println(val);
```

Comment:
Check the assignment's description again, only the FIRST argument should be printed. Not all arguments!